frontend/packages/ceph-storage-plugin/src/components/create-storage-system/external-storage/index.ts

```diff
@@ -0,0 +1,17 @@
+import { ExternalStorage } from './types';
+import { rhcsCanGoToNextStep, ConnectionDetails, rhcsPayload } from './red-hat-ceph-storage';
+import { OCSServiceModel } from '../../../models';
+
+export const SUPPORTED_EXTERNAL_STORAGE: ExternalStorage[] = [
+  {
+    displayName: 'Red Hat Ceph Storage',
```

Comment:
We can perform some validations on these if empty values are passed and do some basic regex checks.

Reply:
Or you could also disable the next button based on these validations for backingstore page. Currently it only checks for ssLoaded and ssErr

Reply:
Its not a dynamic content that can be empty. Its should always have a valid string.
We do not need to tackle such edge cases. If its broken, then user would know , he missed some information or added incorrect.

Reply:
Though I have added null based checks for them wherever required.

Reply:
When i had "IBM Flashsystem" as displayName, i had a 404 error but if we pass an empty value, it is letting me pass through the next steps (which had broken UI and is not ideal for user to do/see) . We should definitely have a basic length check atleast

Reply:
> ![Screenshot from 2021-07-19 15-01-40](https://user-images.githubusercontent.com/57935785/126147902-4a0441db-4b6f-4d0f-847b-e00eab4ab999.png)
> 
> Can we enlarge the heading and add spacing between the title, box and the helper below; the whole design here looks off . Also the next button should ideally be greyed out yet it is letting me pass without any script uploaded.
> 
> Also we can slightly lessen the width for the text area as it is touching the edges of the screen

nit: I would also prefer a slight grey shade for the text area just to separate it from the white background. The text area looks as part of the page itself

Reply:
nit: also add a placeholder on the text area